packages/server/src/errorNormalize.ts

```diff
@@ -66,17 +67,18 @@ export function normalizeAndFormatErrors(
 
       if (originalErrorEnumerableEntries.length > 0) {
         extensions.exception = {
-          ...extensions.exception,
+          ...ensureObject(extensions.exception),
           ...Object.fromEntries(originalErrorEnumerableEntries),
```

Comment:
It's increasing the scope of this PR, but maybe we can remove the behavior of passing all enumerable properties as `extensions.exception`.
This practice can be considered as information leaking, similar to "did you mean".
Now that we gave `formatError` access to the original issue, users can peek and choose what properties they want to put into `extensions`.
So basically, I propose to just get rid of `extensions.exception`.